pkg/virt-launcher/virtwrap/nichotplug.go

```diff
@@ -67,7 +67,7 @@ func hotplugVirtioInterface(vmi *v1.VirtualMachineInstance, converterContext *co
 			return err
 		}
 
-		if err := dom.AttachDevice(strings.ToLower(string(ifaceXML))); err != nil {
+		if err := dom.AttachDeviceFlags(strings.ToLower(string(ifaceXML)), affectLiveAndConfigLibvirtFlags); err != nil {
```

Comment:
the const comes from hostdevice/hotplug, please redefine it here or move it to a common place.

Reply:
? This const comes from https://github.com/kubevirt/kubevirt/blob/c85c9af8cd66a212209320b269989c58ca721322/pkg/virt-launcher/virtwrap/manager.go#L93 

I.e. it is defined "here" afaiu. 
@AlonaKaplan